adapters/adoppler/adoppler.go

```diff
@@ -28,11 +33,18 @@ type adsImpExt struct {
 }
 
 type AdopplerAdapter struct {
-	endpoint string
+	endpoint *template.Template
 }
 
 func NewAdopplerBidder(endpoint string) *AdopplerAdapter {
-	return &AdopplerAdapter{endpoint}
+	e := endpoint + "/processHeaderBid/{{.AdUnit}}"
```

Comment:
Why not include the `/processHeaderBid/{{.AdUnit}}` as part of the endpoint config? It'll give a bit more flexibility for future changes:
```
http://{{.AccountID}}.trustedmarketplace.io/ads/processHeaderBid/{{.AdUnit}}
```

Reply:
The thing is there is other "functions" except `processHeaderBid` and it can be a situation when need to use another on also in the future.

Reply:
Not sure I follow. Are you saying the adapter might to call out to multiple paths in the future based on some properties of the bid request?

Reply:
Yes. I mean there are no such visible plans for that. Just in case. I thought about endpoint in config something more like "base URI". 
I can move `AdUnit` placeholder in config if we need, it is not big difference for me.

Reply:
Gotcha. If it's not a big deal to you, then yes - please move the `AdUnit` bit to the config. This is more in line with the other adapters and gives the host a better idea of the bidder url. If you need to change it in the future as the adapter evolves, no big deal. Let's try to keep it simpler for now.

Looks great otherwise.

Reply:
Added commit with discussed changes.